assets/styles/_fonts.scss

```diff
@@ -0,0 +1,13 @@
+//variable for font- heading (h1) for Introduction slide and Make an impact slide (start-end)
+$redhatdisplay-regular: "Red Hat Display Regular",
```

Comment:
Tenker at variabelnavnet bør si noe om fontens rolle i designet, heller enn å være fontnavnet. F.eks: $font-body eller $font-headings

Reply:
oki, jeg bel litt revet med av ett tips jeg fant når jeg googla ;) * se nederste svar i denne siden(link). ellers er jeg forøvrig enig med deg :)
https://stackoverflow.com/questions/54923441/getting-error-on-importing-custom-fonts-using-sass